components/schemas.yaml

```diff
@@ -93,3 +93,100 @@ Method:
   # Option 2
   type: string
   default: 'TODO: override with own method name'
+
+# ================= Account Abstraction =================
+
+UserOperation:
+  type: object
+  properties:
+    sender:
+      type: string
+      description: The account making the operation
+      default: '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+      pattern: '^0[xX][0-9a-fA-F]+$'
+    nonce:
+      type: integer
+      description: Anti-replay parameter; also used as the salt for first-time account creation
+      default: 1
+    initCode:
+      type: string
+      description: The initCode of the account (needed if and only if the account is not yet on-chain and needs to be created)
+      default: '0x'
```

Comment:
@elanh this field is only needed if the account is not yet on-chain and needs to be created, otherwise it's just `0x`

Most of the time, the value of this field will be `0x`, that's why I left it as that. 

Please let me know if you still want me to change it, would be happy to do so.

Reply:
ah that makes sense, thank u!